src/khoj/routers/email.py

```diff
@@ -47,3 +46,38 @@ async def send_welcome_email(name, email):
             "html": html_content,
         }
     )
+
+
+from pydantic import BaseModel
+
+
+class TextData(BaseModel):
+    uquery: str
+    kquery: str
+    sentiment: str
+
+
+async def feedback(uquery, kquery, sentiment):
```

Comment:
With function naming convention, it's preferred to name it so that the action being completed is super clear just by reading it.

Also adding a parameter for user email.

```suggestion
async def send_query_feedback(uquery, kquery, sentiment, user_email):
```

Reply:
where can I obtain the user's email?